docs/changes.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 ---------
 
+2023-08-11
+^^^^^^^^^^
+
+- Update ``earth_wdmam`` source from version 2.0 to 2.1 [`Choi et al., 2023`_].
```

Comment:
Maybe we can have a more consistent changelog style like:

- ``dataset_name``: Summary of the change [the reference or the issue if any]

```suggestion
- ``earth_wdmam``: Update dataset source from version 2.0 to 2.1 [`Choi et al., 2023`_].
```